resources/analysisTools/qcPipeline/chrom_diff.r

```diff
@@ -1,4 +1,5 @@
 library(getopt)
+library(uuid)
```

Comment:
uuid is not a pre-installed package (also if it happens to be present on the cluster). 

As an alternative you could use `tempfile` from the "base" package, which is guaranteed to be present. 

Also tempfile is actually made for temporary files, so it is 100% certain that no name-clashes occur (I know this is not a strong argument, as UUIDs are considered to be /almost/ 100% safe for this as well).

Reply:
[uuid is not a pre-installed package]: same holds for getopt by the way

Reply:
Right, but getopt was there before. Is "uuid" also used in other scripts of this workflow (like getopt)?

Reply:
no and I do not know the reason for uuid being available despite cluster admins restrict themselves to offer only standard-like packages

Reply:
tempfile() manes use of tempdir(). Can we rely on always getting back a valid path from latter?

Reply:
Of course we can not rely on TEMDIR or so pointing to a useful directory (often it's just too small for the data). 

But, you can set the directory in the tempfile arguments. See `?tempfile`.